### PR TITLE
[RW-1612][RISK=LOW]Recently created/edited concepts does not appear on workspace about "Recently modified list'

### PR DESCRIPTION
**Pull request by NehaBroad**

### CODE REVIEW

File: ui/src/app/views/workspace/component.spec.ts [hunk 1]
```diff
@@ -110,6 +110,7 @@ describe('WorkspaceComponent', () => {
         { provide: BugReportService, useValue: new BugReportServiceStub() },
         { provide: ClusterService, useValue: new ClusterServiceStub() },
         { provide: CohortsService, useValue: new CohortsServiceStub() },
+        { provide: ConceptSetsService },
```
Comment (calbach) [inline ui/src/app/views/workspace/component.spec.ts]: Should this be using the stub?

File: ui/src/testing/stubs/concept-sets-service-stub.ts [hunk 2]
```diff
@@ -30,17 +30,20 @@ export class ConceptSetsServiceStub {
...
-        domain: Domain.MEASUREMENT
+        domain: Domain.MEASUREMENT,
+        lastModifiedTime: new Date().getTime()
       },  {
         id: 347,
         name: 'Mock Concept Set for condition',
         description: 'Mocked for tests',
-        domain: Domain.CONDITION
+        domain: Domain.CONDITION,
+        lastModifiedTime: new Date().getTime() + 8000
```
Comment (calbach) [inline ui/src/testing/stubs/concept-sets-service-stub.ts]: Modified in the future?

Comment (NehaBroad) [reply ui/src/testing/stubs/concept-sets-service-stub.ts]: this was just to manipulate the list for test as it was also dependent on the notebook and cohort list that i did not want to change i had to put the future date

File: ui/src/app/views/workspace/component.spec.ts (hunk 1 above)
Comment (NehaBroad) [reply ui/src/app/views/workspace/component.spec.ts]: This is not needed as          { provide: ConceptSetsService, useValue: new ConceptSetsServiceStub() },
 is already below in the list

File: ui/src/app/views/homepage/component.spec.ts [hunk 3]
```diff
@@ -18,8 +18,10 @@ import {simulateClick, updateAndTick} from 'testing/test-helpers';
 import {ProfileStorageService} from 'app/services/profile-storage.service';
 import {ServerConfigService} from 'app/services/server-config.service';
 import {CohortsService} from 'generated/api/cohorts.service';
+import {ConceptSetsService} from 'generated/api/conceptSets.service';
 import {UserMetricsService} from 'generated/api/userMetrics.service';
 import {WorkspacesService} from 'generated/api/workspaces.service';
+import {ConceptSetsServiceStub} from '../../../testing/stubs/concept-sets-service-stub';
```
Comment (UrsaStutsman) [inline ui/src/app/views/homepage/component.spec.ts]: nit: don't need all the leading ../

**Pull request merged**